modules/apps/callbacks/types/callbacks_test.go

```diff
@@ -228,6 +228,55 @@ func (s *CallbacksTypesTestSuite) TestGetCallbackData() {
 			true,
 			nil,
 		},
+		{
+			"success: source callback with calldata",
```

Comment:
Note: we allow calldata on either side in ibc-go. EVM contract keeper will enforce calldata only set for dest calldata